llvm/lib/CodeGen/AtomicExpandPass.cpp

```diff
@@ -340,7 +342,33 @@ bool AtomicExpand::runOnFunction(Function &F) {
   return MadeChange;
 }
 
-bool AtomicExpand::bracketInstWithFences(Instruction *I, AtomicOrdering Order) {
+bool AtomicExpandLegacy::runOnFunction(Function &F) {
+  if (skipFunction(F))
+    return false;
```

Comment:
This should not use skipFunction, it's a mandatory lowering pass. It's not part of the current code